subscription-manager.spec

```diff
@@ -333,6 +357,7 @@ Obsoletes: python-rhsm <= 1.20.3-1
 A small library for communicating with the REST interface of a Red Hat Unified
 Entitlement Platform. This interface is used for the management of system
 entitlements, certificates, and access to content.
+%endif
```

Comment:
The e.g. description of package subscription-manager-rhsm is there twice. The logic could be IMHO simplified for this package.